src/workerd/api/basics.h

```diff
@@ -578,28 +588,28 @@ class AbortSignal final: public EventTarget {
     }
     JSG_PROTOTYPE_PROPERTY(onabort, getOnAbort, setOnAbort);
     JSG_METHOD(throwIfAborted);
+    JSG_METHOD(skipReleaseForTest);
```

Comment:
Oh, this still needs to be guarded behind "experimental".

Reply:
I'm trying to clean up this test more deeply. I'll make sure any test only methods are protected before I finish.